website/docs/cli/configuration/configuration.mdx

```diff
@@ -24,15 +24,55 @@ Think of this file as where you [bootstrap the settings or configuration of your
+When the `base_path` is not explicitly provided or is empty, Atmos will attempt to infer the appropriate path using the following strategy, in order of precedence:
+1. If the current working directory contains any of the following files or directories: `atmos.yaml`, `.atmos.yaml`, `.github/atmos.yaml`, or `atmos.d/**/*`, Atmos will use the directory containing these files as the `base_path`.
+2. If a Git repository is detected, Atmos uses the root of the Git repository as the `base_path`.
+3. If neither of the above applies, Atmos defaults to the absolute path of `./` as the `base_path`.
 
+Each setting (`components.terraform.base_path`, `components.helmfile.base_path`, `stacks.base_path`, and `workflows.base_path`) is treated independently and can accept either an absolute or a relative path.
+## Configuration File (`atmos.yaml`)
+
+The Atmos (CLI) configuration is loaded from the following locations, in order of precedence, and deep-merged: configurations listed earlier have higher priority, meaning they override any conflicting sections defined in subsequent locations:
+- The `--config` parameter is the relative or absolute path to a valid configuration file or directory containing strictly atmos configuration file(s). It has the highest precedence.
+- The Environment variable `ATMOS_CLI_CONFIG_PATH` (the ENV var should point to a folder without specifying the file name. If it is set, Atmos will only look for the config file in the specified folder). Note, that the `.yaml` and `.yml` extensions are interchangeable and loaded in lexicographical order. Atmos will look for the following files, in order, relative to the `ATMOS_CLI_CONFIG_PATH` environment variable.
+  From the current directory:
+  - `atmos.yaml`
+  - `.atmos.yaml`
+  - `.github/atmos.yaml`
+  - `atmos.d/**/*`
+  - `.atmos.d/**/*`
 - System directory (`/usr/local/etc/atmos/atmos.yaml` on Linux, `%LOCALAPPDATA%/atmos/atmos.yaml` on Windows)
+- `$XDG_CONFIG_HOME/atmos/atmos.yaml` (if `$XDG_CONFIG_HOME` is set)
 - Home directory (`~/.atmos/atmos.yaml`)
-- Current directory (`./atmos.yaml`)
-- Environment variable `ATMOS_CLI_CONFIG_PATH` (the ENV var should point to a folder without specifying the file name)
+- `atmos.d/` directory form base path (if `import` section is not set in the config) For an example, see [`examples/demo-atmos.d`](https://github.com/cloudposse/atmos/tree/main/examples/demo-atmos.d)
+Additionally, Atmos supports `imports` of other CLI configurations. Use imports to break large Atmos CLI configurations into smaller ones, such as organized by top-level section. File imports are relative to the base path (if `import`  section is set in the config). All imports are processed and deep-merged in order, so that the last file in the list supersedes settings in the preceding imports. For an example, see [`examples/demo-atmos-cli-imports`](https://github.com/cloudposse/atmos/tree/main/examples/demo-atmos-cli-imports).
```

Comment:
```suggestion
- `atmos.d/` directory form base path (if `import` section is not set in the config) For an example, see [`examples/demo-atmos.d`](https://github.com/cloudposse/atmos/tree/main/examples/demo-atmos.d)

Additionally, Atmos supports `imports` of other CLI configurations. Use imports to break large Atmos CLI configurations into smaller ones, such as organized by top-level section. File imports are relative to the base path (if `import`  section is set in the config). All imports are processed and deep-merged in order, so that the last file in the list supersedes settings in the preceding imports. For an example, see [`examples/demo-atmos-cli-imports`](https://github.com/cloudposse/atmos/tree/main/examples/demo-atmos-cli-imports).
```